mysql-async/src/main/java/com/github/jasync/sql/db/mysql/codec/MySQLConnectionHandler.kt

```diff
@@ -164,7 +164,9 @@ class MySQLConnectionHandler(
                     }
                     ServerMessage.BinaryRow -> {
                         val m = message as BinaryRowMessage
-                        this.currentQuery!!.addRow(this.binaryRowDecoder.decode(m.buffer, this.currentColumns))
+                        val decodedRow = this.binaryRowDecoder.decode(m.buffer, this.currentColumns)
+                        m.buffer.release()
```

Comment:
my understanding is you don't need to release the buffer from BytesToMessageDecoder